src/app/db/base_class.py

```diff
@@ -0,0 +1,12 @@
+from typing import Any
+from sqlalchemy.ext.declarative import as_declarative, declared_attr
+
+
+@as_declarative()
+class Base:
+    id: Any
```

Comment:
Do we want to have the id ad `Any` for sure?
I could also imagine a `Union` if `int` isn't enough.

Reply:
If we change it to int we enforce, that we must have integers as id everywhere. Guess it's fine for us, but don't see a huge problem to keep `Any` just in case we want to do something different with one model class. IMHO would keep the Any